lib/.eslintrc.yaml

```
@@ -142,6 +142,8 @@ rules:
       message: Use `const { WritableStreamDefaultWriter } = require('internal/webstreams/writablestream')` instead of the global.
     - name: WritableStreamDefaultController
       message: Use `const { WritableStreamDefaultController } = require('internal/webstreams/writablestream')` instead of the global.
+    - name: addEventListener
+      message: Use `FunctionPrototypeCall(EventTarget.prototype.addEventListener, globalThis)` instead of the global.
```

I think the `FunctionPrototypeCall` is not necessary since it would treat `undefined` `this` as the global object anyway.
```suggestion
      message: Use `const { addEventListener } = EventTarget.prototype` instead of the global.
```